pkg/machines/manifest.json.in

```diff
@@ -9,5 +9,6 @@
         "path": "index.html",
         "order": 60
      }
-  }
+  },
+  "content-security-policy": "default-src 'self' 'unsafe-inline' data:"
```

Comment:
What's left that requires unsafe-inline? No other cockpit-shipped page currently needs this, so this requires some justification (at least a comment). Can this be avoided somehow?